src/contracts/core/DelegationManager.sol

```diff
@@ -40,6 +41,12 @@ contract DelegationManager is
     // @dev Maximum Value for `stakerOptOutWindowBlocks`. Approximately equivalent to 6 months in blocks.
     uint256 public constant MAX_STAKER_OPT_OUT_WINDOW_BLOCKS = (180 days) / 12;
 
+    // the number of 12-second blocks in 30 days (60 * 60 * 24 * 30 / 12 = 216,000)
+    uint256 public constant MAX_WITHDRAWAL_DELAY_BLOCKS = 30 days / 12;
```

Comment:
Should be `MAX_WITHDRAWAL_DELAY / 12`, rather than reusing the "30 days" value

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/695/commits/49a2cd544dbf2abd0d498706621422833c8a49ab